lib/internal/url.js

```diff
@@ -402,6 +402,7 @@ ObjectDefineProperties(URL.prototype, {
       };
       const ctx = this[context];
       let ret = ctx.scheme;
+      // https://url.spec.whatwg.org/#url-serializing
```

Comment:
nit: move this one line up, since `ret = ctx.scheme` is step 1 in this algorithm.

Reply:
Moved the comment one line up, thank you!